optuna/integration/mlflow.py

```diff
@@ -156,8 +178,10 @@ def __call__(self, study: optuna.study.Study, trial: optuna.trial.FrozenTrial) -
 
         with mlflow.start_run(
             run_id=trial.system_attrs.get(RUN_ID_ATTRIBUTE_KEY),
-            run_name=str(trial.number),
-            nested=self._nest_trials,
+            experiment_id=self._mlflow_kwargs.get("experiment_id"),
```

Comment:
The current test cases do not specify `experiment_id` in the `mlflow_kwargs` argument. Is it possible to add a test case to set `experiment_id`?

Reply:
Done in eed521106358dd5ce7fddae56501602959d63bb4.